.gitignore

```diff
@@ -1,2 +1,3 @@
 streamlit_authenticator/__pycache__
 /test
+venv
```

Comment:
I'm not sure if this is meant to be here?

Reply:
Yeah, I added it for local development. Many developers user `venv/` for their virtual environment.